iphone/Classes/ContactsModule.m

```diff
@@ -716,12 +719,16 @@ -(TiContactsPerson*)createPerson:(id)arg
 		}
 		NSError *error = nil;
 		CNMutableContact *newContact = [[CNMutableContact alloc] init];
-		TiContactsPerson* newPerson = [[[TiContactsPerson alloc] _initWithPageContext:[self executionContext] contactId:newContact module:self] autorelease];
+		TiContactsPerson* newPerson = [[[TiContactsPerson alloc] _initWithPageContext:[self executionContext]
```

Comment:
We may should consider to leave the old initializer in place, because module developers might use the interface their modules. Also, we don't need to nil the observer on those.